### ISSUE TITLE
combine dev and stage bash scripts

### ISSUE THREAD

**Issue by jaykay12** [jaykay12/Auth-API#23]
Currently having 2 separate bash scripts for running app on dev and stage environment.
- Convert them into single `start.sh`
- Use flag valued like `--stage` and `--dev` (`-s` and `-d`)
-Make `--dev` as the default behaviour
- Update builder guide accordingly

**Comment by jaykay12** [comment]
Resolved by 6650508c053ef38b6f80469b17256069988694f2

---

### Running API on DEV Environment
```console
$ bash start.sh --dev
 * Serving Flask app "api" (lazy loading)
 * Environment: development
 * Debug mode: on
```
```console
$ bash start.sh -d
 * Serving Flask app "api" (lazy loading)
 * Environment: development
 * Debug mode: on
```

---

### Running API on STAGE Environment
```console
$ bash start.sh --stage
 * Serving Flask app "api" (lazy loading)
 * Environment: stage
 * Debug mode: off
```
```console
$ bash start.sh -s
 * Serving Flask app "api" (lazy loading)
 * Environment: stage
 * Debug mode: off
```

---

### Default behaviour is --dev
```console
$ bash start.sh
 * Serving Flask app "api" (lazy loading)
 * Environment: development
 * Debug mode: on
```

**Comment by jaykay12** [comment]
Builder guide updated by b8d57f43146962c8057db2457811a6a5599f3345

**Comment by jaykay12** [comment]
Closed by #25
